MIDIchlorians/MIDIchlorians/AnimationSequence.swift

```diff
@@ -45,4 +45,65 @@ class AnimationSequence {
         tickCounter += 1
         return array
     }
+
+    func getJSON() -> String? {
+        var arrayOfStrings = [[String]?]()
```

Comment:
maybe call this something like, animationSequence? Cos it's an animation sequence JSON in string form right?